ui/app/js/app.js

```diff
@@ -266,8 +266,8 @@ angular.module('am.controllers').controller('SilenceCtrl',
   }
 );
 
-angular.module('am.controllers').controller('SilencesCtrl', function($scope, $location, Silence, silences) {
-  $scope.totalItems = silences.data.totalSilences;
+angular.module('am.controllers').controller('SilencesCtrl', function($scope, $location, Silence, totalSilences) {
+  $scope.totalItems = totalSilences;
```

Comment:
Discussed with grobie in person. This value is injected from the `$routeProvider`. The code has been fixed so that it properly waits for the promise in the router to resolve before loading the controller.